### PR TITLE
Add support for Second Generation Intuos Pro Small

### PR DESCRIPTION
**Pull request by skomra**
Also parallel the work done for the Intuos Pro Large in commit aee1636706c0 for the Intuos Pro Medium.

Signed-off-by: Aaron Armstrong Skomra <aaron.skomra@wacom.com>

### CODE REVIEW

File: data/intuos-pro-2-s.tablet [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+[Device]
+Name=Wacom Intuos Pro S
+DeviceMatch=usb:056a:0392;bluetooth:056a:0393;
+Class=Intuos5
+Width=6
+Height=4
+Layout=intuos-pro-2-s.svg
+IntegratedIn=
+Styli=0x842;0x84a;0x160802;0x16080a;0x100802;0x10080a;0x140802;0x14080a;0x120802;0x100804;0x10080c;0x100902;0x10090a;0x80842;0x100842;0x10084a
```
Comment (whot) [inline data/intuos-pro-2-s.tablet]: 23e33ffbe717063bf537cfc38b002e52443759f7 started sorting all the IDs in numerical order, can you do the same here please?

File: data/intuos-pro-2-s.tablet [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Wacom
+# Intuos Pro 2 S
+# PTH-460
```
Comment (whot) [inline data/intuos-pro-2-s.tablet]: This needs to (also) go into the new `ModelName` property, see b94cd3fcf97b3297943032e0f5b29e7cfbd29828

**Comment by skomra** [comment]
I "re-requested review" after making the requested changes. I hope that's the right procedure.

**Comment by whot** [comment]
I think that's the correct procedure, but anything that ends up in my inbox will do ;) Thanks

**Pull request merged**